docs/BasicCommunication/OnSystemRequest/index.md

```diff
@@ -56,6 +60,9 @@ BC.OnSystemRequest in "Proprietary" Policy Table Update Flow
 ![Proprietary PTU](./assets/OnSystemRequest_in_Proprietary_PTU_flow.png)
 |||
 
+_EXTERNAL proprietary_ Policy Table Update Flow
+![EXTERNAL proprietary](https://github.com/DrachenkoAnastasiia/sdl_hmi_integration_guidelines/blob/PTU_external_proprietary/docs/BasicCommunication/PolicyUpdate/assets/diagram_PolicyUpdate_external_proprietary.png) 
```

Comment:
Diagram links should to the GitHub repository directly, they should be relative links like the others in the document.

In addition, they should be surrounded like this 

```
|||
Content
![alt text](assets/diagram.jpg)
|||
```

as specified in the [SDL Markdown Spec](https://github.com/smartdevicelink/sdl_markdown_spec). This formats it in a way that the diagram can be opened and shown on demand, rather than taking up a large section of the page by default.